x-pack/plugins/aiops/public/components/change_point_detection/use_change_point_agg_request.ts

```diff
@@ -118,10 +121,12 @@ export function useChangePointResults(
   fieldConfig: FieldConfig,
   requestParams: ChangePointDetectionRequestParams,
   query: QueryDslQueryContainer,
-  splitFieldCardinality: number | null
+  splitFieldCardinality: number | null,
+  embeddingOrigin: string = AIOPS_TELEMETRY_ID.AIOPS_DEFAULT_SOURCE
```

Comment:
I reckon that instead of passing it as an argument, it should be retrieved from the context.

Reply:
@alvarezmelissa87 you can add it to `AiopsAppContext`

Reply:
Moved to AiopsAppContext in 276d6ec28623b5447c984da6d31bf9fd939252d6